agent/provider/src/market/negotiator/builtin/manifest.rs

```diff
@@ -90,27 +85,36 @@ impl From<PolicyConfig> for ManifestSignature {
 
         ManifestSignature {
             enabled,
-            trusted_keys: config.trusted_keys.unwrap_or_default(),
+            keystore: config.trusted_keys.unwrap_or_default(),
         }
     }
 }
 
+impl ManifestSignature {
+    /// Verifies fields base64 encoding, then validates certificate, then validates signature, then verifies manifest content and returns it
+    fn verify_signature<S: AsRef<str>>(
+        &self,
+        demand: &ProposalView,
+        manifest: S,
+    ) -> anyhow::Result<()> {
+        let sig: String = demand.get_property(DEMAND_MANIFEST_SIG_PROPERTY)?;
+        log::trace!("sig_hex: {}", sig);
+        let sig_alg: String = demand.get_property(DEMAND_MANIFEST_SIG_ALGORITHM_PROPERTY)?;
+        log::trace!("sig_alg: {}", sig_alg);
```

Comment:
Do we still support these fields?

Reply:
Yes, because there can be various digest algorithms. The reason it is still a `String` is that rust `openssl` library does not provide any list of available options as a code (enum, types). As far as I remember original `openssl` lib can list digest algorithms through function.

Reply:
Ok, sorry. I thought that we have only `cert` field containing all necessary info, but I was wrong. Merging